tasks/publish.js

```diff
@@ -36,8 +35,6 @@ const publishRelease = ({ notes, tag, token }) =>
   })
 
 async function main() {
-  await build.buildAllPlatforms()
```

Comment:
How is this program going to publish a release on GitHub it's not longer building it?

Reply:
As of: https://github.com/cosmos/voyager/issues/1059#issuecomment-416956483

Reply:
I see.